src/utils.ts

```diff
@@ -43,7 +43,9 @@ const addHeadOverToGitPOAP = (comment: string): string =>
 
 // Hardcode extra comments for ethereum-org-website
 const ethereumOrgWebsiteComment =
-  'Be sure to join the [Ethereum.org discord](https://ethereum.org/discord) if you are interested in contributing further to the project or have any questions for the team.';
+  'Join the [ethereum.org Discord server](https://ethereum.org/discord) to explore more ways to contribute to the project. Depending on the tasks you complete, you may also unlock additional badges (OATs). Visit [ethereum.org/contributing](https://ethereum.org/contributing) to learn more.';
```

Comment:
I'd rather not advertise other badge platforms through the GitPOAP bot - do you mind changing it just to "you may also unlock additional badges" or " ... additional rewards"?

Reply:
Can change this for sure!